lib/redmine_issue_trash/attachment_patch.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module RedmineIssueTrash
+  module AttachmentPatch
+    module Prepend
+      module ClassMethods
+        # app/views/trashed_issues/show.html.erb:66
+        #   <%= textilizable @issue, :description, :attachments => @trashed.attachments %>
+        # @trashed.attachments(TrashedIssue::IssueWrapper#attachments)に#idがAttachmentを使っているため
+        def latest_attach(attachments, filename)
+          super(attachments.reject { |attachment| attachment.id.blank? }, filename)
```

Comment:
[imo]
細かいのですが， `reject...blank?` だと否定で否定になるため， `select...present?` ではいかがでしょうか．

```suggestion
          super(attachments.select { |attachment| attachment.id.present? }, filename)
```

Reply:
[imo]
Attachment#idが空文字列や空配列，空Hashなどになることはあるでしょうか．もしもIntegerかnilのみであれば次のようにも書けそうに思いました．

```suggestion
          super(attachments.select(&:id), filename)
```

Reply:
Attachment#latest_attachを使っている既存のコードへの影響がないかを考えましたが，idがnilのときは https://github.com/redmine/redmine/blob/6.0.5/app/models/attachment.rb#L369 で今回の実行時例外が発生するため，なさそうと理解しました！